code/modules/client/preference_setup/general/04_equipment.dm

```diff
@@ -26,8 +26,8 @@
 	pref.all_underwear =          R.read("all_underwear")
 	pref.all_underwear_metadata = R.read("all_underwear_metadata")
 	pref.backpack_metadata =      R.read("backpack_metadata")
-	pref.starting_cash_choice =   R.read("starting_cash_choice")
-	pref.survival_box_choice =    decls_repository.get_decl_by_id(R.read("survival_box"), validate_decl_type = FALSE)
+	pref.starting_cash_choice =   decls_repository.get_decl_by_id_or_var(R.read("starting_cash_choice"), /decl/starting_cash_choice)
+	pref.survival_box_choice =    decls_repository.get_decl_by_id_or_var(R.read("survival_box"), /decl/survival_box_option)
```

Comment:
I'm not totally sure if `get_decl_by_id_or_var` is valid here, was it serialized by name beforehand? Either way, at worst it's just wasted effort, not going to harm anything.